### PR TITLE
feature: PostHog popups react demo

### PR DESCRIPTION
**Pull request by lharries**
Adds examples of how to use PostHog JSON feature flags for creating popups. 

See the Readme within this branch for more information

### CODE REVIEW

**Comment by lharries** [comment]
@kappa90 would be great to get your eyes on this before merging it

File: react/src/Popup.js [hunk 1]
```diff
@@ -0,0 +1,154 @@
+import React, { useState, useEffect } from 'react'
+
+import './Popup.css'
+
+import EXAMPLE_FLAG_POPUP from './example-flag-popup.json'
```
Comment (ivanagas) [inline react/src/Popup.js]: When you copy this file to another project, this import causes an error.

File: react/src/Popup.js [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+    const [payload, setPayload] = useState({})
+    const [locationCSS, setLocationCSS] = useState({})
+    const [activeFlag, setActiveFlag] = useState(null)
+    const posthog = usePostHog()
+    const activeFlags = useActiveFeatureFlags()
+
+    useEffect(() => {
+        if (DEBUG_SHOW_LOCAL_POPUP) {
+            initPopUp(
+                EXAMPLE_FLAG_POPUP,
```
Comment (ivanagas) [inline react/src/Popup.js]: If you remove the earlier import, should remove this mention too.

**Review by ivanagas — COMMENTED** [review]
1. Should this repo by named "posthog-popups" ? 
2. Is it in the plans to turn into site app or publish on npm?

Will wait for this to get settled before finishing this tutorial: https://github.com/PostHog/posthog.com/pull/5488

File: react/src/Popup.js (hunk 1 above)
Comment (lharries) [reply react/src/Popup.js]: done

File: react/src/Popup.js (hunk 2 above)
Comment (lharries) [reply react/src/Popup.js]: done

**Comment by lharries** [comment]
Going to merge it. I'm confident with the code and we are using something very similar in product which has been reviewed by Em

**Pull request merged**